### ISSUE TITLE
Could Glaux OS fit on a floppy?

### ISSUE THREAD

**Issue by informer2016** [glauxosdever/glaux-os-third#1]
Even today the floppies are still being used, for example - as virtual floppies inside the coreboot open source BIOS. Just imagine: your wonderful OS could be a part of someone's BIOS build! _(for coreboot supported motherboard, maybe you have or could get one - see https://www.coreboot.org/Supported_Motherboards )_

If you already have a coreboot-supported motherboard, or a real chance to get one, - wouldn't it be cool to be able to launch your own OS straight from the BIOS chip? ;) With one simple command its possible to add any floppy to coreboot BIOS build - and then you see it as a boot entry! Multiple floppies could be added this way _(as long as you have enough space left inside the BIOS flash chip, luckily LZMA compression could be used for the stored floppies to reduce their occupied size)_

**Comment by glauxosdever** [comment]
It's been observed that you have opened several similar issues in hobbyist OS projects.

glaux-os-third, however, is NOT an active project anymore, since it meant to be replaced by an OS of different design at some point in the future. Suggestions are always welcome, as long as they are reasonable.

To elaborate, floppies are obsolete, slow and unreliable media with too little capacity and no one would benefit from supporting them, even more since glaux-os-third can't do anything useful (and won't be able to do anything useful because it is not an active project anymore). So this suggestion is not reasonable.

Furthermore, are you saying glaux-os-third could be part of a BIOS build? What would be the purpose of this?

In short, you are commenting on an abandoned project that doesn't do anything useful, trying to persuade me to add floppy support (as seen from what you commented on other projects) and to consider making it part of a BIOS build.

**Comment by informer2016** [comment]
@glauxosdever Sorry, not a bot here, its' just I did not see any need to rewrite my message "to be unique". I am looking for the best floppy based operating systems: after the replacement of bloated proprietary UEFI with that slim open source BIOS - now there is a lot of free space (whole 3MB! :wink: ) that would be great to occupy with something useful or maybe just for fun 

I realize that floppies are the outdated format, and to me its' only reasonable to use them as "virtual floppies" built in inside this BIOS - just because the other formats like "tiny hdd img" are not supported there... Not trying to persuade you or something, just maybe in some cases I might haven't noticed if its already possible to build such a floppy _(e.g. few lines in a hidden Makefile)_ - and I'm trying to clarify for the projects I got interested in - maybe its already done, or maybe it can't be done even if I try because of great OS size

Wish you a great 2018 year of Yellow Earth Dog, happy life, bugless code, <br/>and good luck to your coming OS projects regardless of their floppy support :smiley:
